src/app/pages/common/pages-not-found-page/pages-not-found-page.component.scss

```diff
@@ -0,0 +1,34 @@
+@use 'sizes';
+
+.not-found {
+    min-height: calc(100vh - #{sizes.$navbar-height});
+    padding-top: 80px;
+    
+    @media only screen and (min-width: 900px) {
+        padding-top: 0px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+    }
+}
+
+@media only screen and (min-width: 900px) {
+    h1 {
+        font-size: 40px;
```

Comment:
Lets go with much bigger size for h1 in that case. Lets go for 80px for h1, increase padding bottom to 25px and also you can have same font-sizes and padding for mobile and desktop.